docs/general/ad-filtering/create-own-filters.md

```diff
@@ -331,7 +331,23 @@ Rules with `$denyallow` modifier are not supported by AdGuard for iOS, Safari, a
 
 #### **`$domain`** {#domain-modifier}
 
-`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). To add multiple domains to one rule, use the `|` character as a separator.
+`$domain` limits the rule scope to requests made **from** the specified domains and their subdomains (as indicated by the [Referer](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) HTTP header). 
+
+###### Syntax
+
+The modifier is a list of one or more expressions separated by `|` symbol, each of which is matched against a domain in a certain way depending on its type (see below).
+
+```
+ domains = ["~"] entry_0 ["|" ["~"] entry_1 ["|" ["~"]entry_2 ["|" ... ["|" ["~"]entry_N]]]]
+ entry_i = ( regular_domain / any_tld_domain / regexp )
+```
+
+* **`regular_domain`** — a regular domain name (`domain.com`). Corresponds the specified domain and its subdomains. It is matched lexicographically.
+* **`any_tld_domain`** — a domain name ending with wildcard character as TLD (`domain.*`).  Corresponds to the specified domain and its subdomains with any TLD. It is matched lexicographically.
```

Comment:
Actually, the wildcard matches eTLD (effective TLD), not just TLD, at least in CoreLibs. For instance, in `example.co.uk` `co.uk` is eTLD and `uk` is TLD. Not sure if it should be described explicitly but I would propose to fix the term.

@slavaleleka is it true for tsurlfilter?

Reply:
for `example.*` in tsurlfilter we check the [public suffix](https://publicsuffix.org/learn/) which is `co.uk` for `example.co.uk`

Reply:
It is the same. So, any of them can be used here.